README.md

```diff
@@ -27,18 +27,91 @@ We are continuously adding more functionality to go-mysql-server. We support a s
 
 |                        |                                     Supported                                     |
 |:----------------------:|:---------------------------------------------------------------------------------:|
-| Comparison expressions | !=, ==, >, <, >=,<=, BETWEEN                                |
-| Null check expressions | IS NULL, IS NOT NULL                               |
-|  Grouping expressions  | COUNT, MIN, MAX ,AVG                                  |
-|  Standard expressions  | ALIAS, LITERAL, STAR (*)                             |
-| Statements       | CROSS JOIN, INNER JOIN, DESCRIBE, FILTER (WHERE), GROUP BY, LIMIT, SELECT, SHOW TABLES, SORT, DISTINCT  |
-| Functions | SUBSTRING |
+| Comparison expressions | !=, ==, >, <, >=,<=, BETWEEN, REGEXP, IN, NOT IN |
+| Null check expressions  | IS NULL, IS NOT NULL |
+| Grouping expressions | COUNT, MIN, MAX ,AVG |
+| Standard expressions  | ALIAS, LITERAL, STAR (*) |
+| Statements  | CROSS JOIN, INNER JOIN, DESCRIBE, FILTER (WHERE), GROUP BY, LIMIT/OFFSET, SELECT, SHOW TABLES, SORT, DISTINCT, CREATE TABLE, INSERT |
+| Functions | SUBSTRING, ARRAY_LENGTH |
 | Time functions | YEAR, MONTH, DAY, HOUR, MINUTE, SECOND, DAYOFYEAR |
 
 ## Custom functions
 
 - `IS_BINARY(blob)`: returns whether a BLOB is a binary file or not
 
+## Example
+
+`go-mysql-server` has a sql engine and a server implementation, so to start a server you must instantiate the engine and give it your `sql.Database` implementation that will be in charge to handle all the logic about retrieving the data from your source :
+
+```go
+...
+driver := sqle.New()
+driver.AddDatabase(createTestDatabase())
```

Comment:
maybe add to he readme small `createTestDatabase` function, so someone can just copy-paste and compile (instead of looking into examples)